### ISSUE TITLE
create-repos.py --add-students uses now-defunct _gitlab_session cookie

### ISSUE THREAD

**Issue by GabrielDougherty** [GabrielDougherty/edinboro-gitlab-assignment#13]
Running `./create-repos.py --students gd160206 --add-students --token-file test_token gabriel-class` results in the following prompt:

```
You want students to be added to their project/repository.
This script adds students by interfacing with the codestore.cs.edinboro.edu website directly.
Please login to http://codestore.cs.edinboro.edu and enter your _gitlab_session cookie from codestore.cs.edinboro.edu below.
codestore.cs.edinboro.edu _gitlab_session cookie value:
```

Authentication with the cookie has [been deprecated](https://gitlab.com/gitlab-org/gitlab-ce/issues/31077)

In place of this:
- add students using the API and the token we already have
- Can go off of API description [here](https://docs.gitlab.com/ee/api/members.html#add-a-member-to-a-group-or-project)

This affects lines around [here](https://github.com/GabrielDougherty/edinboro-gitlab-assignment/blob/140a757bb9cd26d5d5cb3f74834915c94e9f80de/create-repos.py#L40) and [here](https://github.com/GabrielDougherty/edinboro-gitlab-assignment/blob/140a757bb9cd26d5d5cb3f74834915c94e9f80de/create-repos.py#L40)

**Comment by GabrielDougherty** [comment]
I now see that the code actually uses `_gitlab_session_cookie` to send an invite email if the user doesn't exist already. The user is added to the repo when they click the invite link.

Unfortunately, this cannot be accomplished with the API, see [here](https://gitlab.com/gitlab-org/gitlab-ce/issues/33626)
So, we will need to change the core functionality. I propose:
- Create user, if they don't already exist
- Add user to repository using API
- Then, email them to inform they have been added

**Comment by GabrielDougherty** [comment]
Fixed in 2356da27bbd3dd33630ccedd235400948a21789e
